init.lua

```diff
@@ -222,3 +222,31 @@ minetest.register_chatcommand("whereis", {
 				pos.x, pos.y, pos.z)
 	end
 })
+
+local staff = {}
+minetest.register_chatcommand("s", {
+	params = "<msg>",
+	description = "Send a message on the staff channel",
+	privs = { ban = true },
```

Comment:
Check for `kick` and `ban` here:

```suggestion
	privs = { kick = true, ban = true },
```